tileview/aar-release.gradle

```diff
@@ -13,6 +13,7 @@ task androidJavadocs(type: Javadoc) {
     source = android.sourceSets.main.java.srcDirs
     ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
     classpath += files(ext.androidJar)
+    failOnError = false
```

Comment:
These are the error I get on JitPack. I also get them on a local build btw. 
https://jitpack.io/com/github/tsuijten/TileView/-2.1.7-g79c78de-2/build.log

I'll create a new issue for that and remove the commit from my PR.